src/app/modules/shared-components/components/path-suggestion/path-suggestion.component.ts

```diff
@@ -29,8 +36,8 @@ export class PathSuggestionComponent implements OnDestroy, OnChanges {
   state$ = this.itemId$.pipe(
     switchMap(itemId => this.getBreadcrumbsFromRootsService.get(itemId).pipe(
       map(group =>
-        (group.length > 0 ? group.map(breadcrumbs =>
-          breadcrumbs.slice(0, breadcrumbs.length - 1).map(item => ({
+        (group.length > 0 ? group.filter(breadcrumbs => breadcrumbs.length > 1).map(breadcrumbs =>
```

Comment:
Is this filter needed? Isn't it just covered by the general case?

Reply:
The response coming with hovered item, in previous version we did hide it (now it's logo). But should we show just logo in path when response has just 1 item? Example:

![image](https://user-images.githubusercontent.com/7044736/220091823-11f348ae-53ee-428a-a3e4-4a7f17fc521b.png)

Reply:
It means it is one of your roots. Yes, you shouldn't hide it, definitely. Now, the question is whether it is clear... 
Actually the idea of the icon itself is not clear at all when I see it in practice... could you just replace this icon by the name of the destination so `Skill #3` in this case?
Sorry for the change.

Reply:
Sure, Fixed